crates/nostr/src/event/tag.rs

```diff
@@ -256,6 +256,39 @@ where
     }
 }
 
+/// Unchecked Relay Url
+#[derive(Debug, Clone, Eq, PartialEq, PartialOrd, Ord)]
+pub struct UncheckedRelayUrl(String);
+
+impl FromStr for UncheckedRelayUrl {
+    type Err = Error;
+
+    fn from_str(s: &str) -> Result<Self, Self::Err> {
+        Ok(UncheckedRelayUrl(s.to_owned()))
```

Comment:
Can you change this to `Ok(Self(s.to_owned()))`?

The same in the other pieces of code that you added :)

Sorry for nitpicking

Reply:
please nitpick still learning rust so its helpful

Reply:
Ahaha, okay :)